src/components/TextArea/TextArea.jsx

```diff
@@ -0,0 +1,154 @@
+import React, { useState, forwardRef } from 'react'
+import styled, { css } from '@xstyled/styled-components'
+import { variant } from '@xstyled/system'
+import PropTypes from 'prop-types'
+
+import { Flex, Box } from '../Grid'
+import { Caption, Typography } from '..'
+
+const TextArea = forwardRef(({ label, caption, placeholder, disabled, name, ...props }, ref) => {
+  const [focus, setFocus] = useState(false);
+
+  return (
+    <Wrapper disabled={disabled} {...props}>
+      <Label forwardedAs='label'>
+        {label}
+        <Container focus={focus}>
+          <Input
+            ref={ref}
+            name={name}
+            placeholder={placeholder}
+            onFocus={() => setFocus(true)}
+            onBlur={() => setFocus(false)}
+            disabled={disabled}
+            {...props}
+          />
+        </Container>
+      </Label>
+      <CaptionMessage>{caption}</CaptionMessage>
+    </Wrapper>
+  )
+});
+
+const errorVariant = variant({
+  prop: 'error',
+  default: false,
+  variants: {
+    true: css`
+      label,
+      p {
+        color: error;
+      }
+      div {
+        border-color: error;
+        span {
+          color: error;
+        }
+      }
+    `,
+    false: css``
+  }
+})
+
+const disabledVariant = variant({
+  prop: 'disabled',
+  default: false,
+  variants: {
+    true: css`
+      p {
+        color: disabled;
+      }
+      div {
+        border-color: gray.400;
+        background-color: gray.100;
+        span {
+          color: gray.400;
+        }
+      }
+    `,
+    false: css``
+  }
+})
+
+const focusVariant = variant({
+  prop: 'focus',
+  default: false,
+  variants: {
+    true: css`
+      border-color: blue.50;
+      border-width: 2px;
+      padding: 0;
+    `,
+    false: css``
+  }
+})
+
+const Wrapper = styled(Box)`
+  ${errorVariant};
+  ${disabledVariant};
+`
+
+const Label = styled(Typography)`
+  display: block;
+  font-size: 2;
+  line-height: 1;
+  font-weight: 1;
+`
+
+const Container = styled(Flex)`
+  border: 1px solid black;
+  border-radius: 2;
+  padding: 1px;
+  min-height: 1;
+  margin: 3 0 2 0;
+  height: 90px;
+  ${focusVariant}
+`
+
+const Input = styled.textarea`
+  border: none;
+  resize: none;
+  font-size: 3;
+  line-height: 3;
+  padding: 3;
+  background-color: transparent;
+  font-family: 'Open Sans', sans-serif;
+  height: calc(100% - 16px);
+  width: 100%;
+
+  &::placeholder {
+    color: gray.600;
+  }
+
+  &:focus {
+    outline: none;
+  }
+
+  &:disabled {
+    ::placeholder {
+      color: disabled;
+    }
+  }
+`
+
+const CaptionMessage = styled(Caption)`
+  line-height: 1;
+  font-size: 1;
+`
+
+TextArea.defaultProps = {
+  label: undefined,
```

Comment:
Acho que é interessante passar uma string como defaultProps no label, porque pelo Figma, sempre deverá ter um label.

Reply:
Não ficou claro pra mim se realmente vai ter sempre um `label`, acho melhor confirmarmos com alguém do design, pois pensando a nível de um Design System, isso certamente limitaria o componente. Então se for o caso, passar uma string no `defaultProps` não faria sentido, já que sempre vai ter, colocariamos como `required`. O que tu acha?

Reply:
Acho que realmente faz sentido teu ponto. É bom confirmar com alguém sim!

Reply:
Troquei uma idéia com a Yasmin e não temos mapeados casos onde o label não exista, ela deixou ao meu critério, nesse caso vou seguir o que já vem sendo feito no projeto e deixar a propriedade como obrigatória.

Reply:
Eu acho que não existe a necessidade de colocar `undenifed` como defaultProps, visto que toda props tem esse valor como default implicitamente.

Reply:
Beleza, irei remover então, eu costumava utilizar assim para aparecer essa informação no styleguidist, vou conferir se implicitamente aparece no storybook, pois é a primeira vez que estou utilizando-o.

Reply:
Acredito que deveria ser passado só o default props do disabled.